graphula-core/src/Graphula.hs

```diff
@@ -279,26 +267,26 @@ runGraphulaIdempotentT action = mask $ \unmasked -> do
 
 
 newtype GraphulaLoggedT m a =
-  GraphulaLoggedT {runGraphulaLoggedT' :: ReaderT (IORef (Seq Value)) m a}
-  deriving (Functor, Applicative, Monad, MonadIO, MonadReader (IORef (Seq Value)))
+  GraphulaLoggedT {runGraphulaLoggedT' :: ReaderT (IORef (Seq Text)) m a}
+  deriving (Functor, Applicative, Monad, MonadIO, MonadReader (IORef (Seq Text)))
 
 instance MonadTrans GraphulaLoggedT where
   lift = GraphulaLoggedT . lift
 
 instance MonadIO m => MonadGraphulaBackend (GraphulaLoggedT m) where
-  type Logging (GraphulaLoggedT m) = ToJSON
+  type Logging (GraphulaLoggedT m) = Show
   type Generate (GraphulaLoggedT m) = Arbitrary
   generateNode = liftIO $ generate arbitrary
   logNode n = do
     graphLog <- ask
-    liftIO $ modifyIORef' graphLog (|> toJSON n)
+    liftIO $ modifyIORef' graphLog (|> pack (show n))
```

Comment:
`Seq` is lazy in its elements so we're not really running `pack . show` until (or unless) the sequence is written to a file.